apps/web/app/(app)/[emailAccountId]/assistant/onboarding/CategoriesSetup.tsx

```diff
@@ -40,6 +43,147 @@ import { Checkbox } from "@/components/Checkbox";
 
 const NEXT_URL = "/assistant/onboarding/draft-replies";
 
+function DigestCheckbox({
+  value,
+  onChange,
+}: {
+  value: CategoryAction;
+  onChange: (value: CategoryAction) => void;
+}) {
+  const isDigest = value === "label_digest" || value === "label_archive_digest";
```

Comment:
these options arent relevant for digest.

if you want to do an action of:
archive, label, digest you'd just add 3 actions to a rule:
![CleanShot 2025-05-27 at 11 00 44@2x](https://github.com/user-attachments/assets/04be29ca-a271-40ba-ac63-775ce41a05a5)

cold email blocker works differently.